docs/source/models/gbd2017/causes/diarrheal_diseases.rst

```diff
@@ -62,9 +62,128 @@ I: _I_nfected and currently experiencing a diarrheal disease bout
 Data Descriptions
 -----------------
 
-.. todo::
+.. list-table:: State Definitions
+	:widths: 5 10 10
+	:header-rows: 1
+	
+	* - State
+	  - State name
+	  - Definition
+	* - S
+	  - **S**\ usceptible
+	  - Simulant currently has diarrheal disease
+	* - I
+	  - **I**\ nfected
+	  - Simulant does not currently have diarrheal disease
+
+.. list-table:: State Data
+	:widths: 5 10 10 20
+	:header-rows: 1
+	
+	* - State
+	  - Measure
+	  - Value
+	  - Notes
+	* - I
+	  - prevalence
+	  - prevalence_c302
+	  -
+	* - I
+	  - birth prevalence
+	  - 0
+	  - 
+	* - I
+	  - excess mortality rate
+	  - :math:`\frac{\text{deaths_c302}}{\text{population} \,\times\, \text{prevalence_c302}}`
+	  -
+	* - I
+	  - disability weight
+	  - :math:`\displaystyle{\sum_{s\in \text{sequelae_c302}}} \scriptstyle{\text{disability_weight}_s \,\times\, \text{prevalence}_s}`
+	  -
+	* - S
+	  - prevalence
+	  - 1-prevalence_c302
+	  -
+	* - S
+	  - birth prevalence
+	  - 1
+	  - 
+	* - S
+	  - emr
+	  - 0
+	  -
+	* - S
+	  - disability weight
+	  - 0
+	  -
+	* - All
+	  - cause-specific mortality rate
+	  - :math:`\frac{\text{deaths_c302}}{\text{population}}`
+	  -
+
+.. list-table:: Transition Data
+	:widths: 10 10 10 10 10
+	:header-rows: 1
+	
+	* - Transition
+	  - Source State
+	  - Sink State
+	  - Value
+	  - Notes
+	* - i
+	  - S
+	  - I
+	  - :math:`\frac{\text{incidence_c302}}{1-\text{prevalence_c302}}`
+	  - We transform incidence to be a rate within the susceptible population.
+	* - r
+	  - I
+	  - S
+	  - remission_m1181
+	  - Already a rate within with-condition population
+
+	  
+.. list-table:: Data Sources and Definitions
+	:widths: 1 3 10 10
+	:header-rows: 1
+	
+	* - Value
+	  - Source
+	  - Description
+	  - Notes
+	* - prevalence_c302
+	  - como
+	  - Prevalence of diarrheal diseases
+	  -
+	* - deaths_c302
+	  - codcorrect
+	  - Deaths from diarrheal diseases
+	  -
+	* - incidence_c302
+	  - como
+	  - Incidence of diarrheal disease within the entire population
+	  - 
+	* - remission_m1181
+	  - dismod
+	  - Remission of diarrheal disease within the infected population
+	  -
+	* - population
+	  - demography
+	  - Mid-year population for given age/sex/year/location
+	  -
+	* - sequelae_c302
+	  - gbd_mapping
+	  - List of 4 sequelae for diarrheal diseases
+	  - Note Guillain-Barre due to diarrheal diseases is included in sequelae.
+	* - prevalence_s{`sid`}
+ 	  - como
+	  - Prevalence of sequela with id `sid`
+	  -
+	* - disability_weight_s{`sid`}
+	  - gbd_mapping
```

Comment:
Can we get the actual disability weights from gbd_mapping, or do we have to look those up somewhere else (e.g. the YLD Appendix)?